src/coreComponents/mesh/simpleGeometricObjects/Cylinder.hpp

```diff
@@ -72,8 +72,8 @@ class Cylinder : public SimpleGeometricObjectBase
 
   struct viewKeyStruct
   {
-    static constexpr char const * point1String() { return "point1"; }
-    static constexpr char const * point2String() { return "point2"; }
+    static constexpr char const * point1String() { return "lowerFaceCenter"; }
+    static constexpr char const * point2String() { return "upperFaceCenter"; }
```

Comment:
@sytuannguyen Are you sure that there is the concept of "lower" and "upper" face in the `Cylinder` class? Upper and lower with respect to what?

Reply:
@francoishamon You are right. How about firstFaceCenter and secondFaceCenter?

Reply:
Yes, I think this is better

Reply:
done